rskj-core/src/main/java/co/rsk/remasc/Remasc.java

```diff
@@ -199,7 +202,19 @@ RskAddress getRskLabsAddress() {
     }
 
     private Coin payToFederation(Constants constants, boolean isRskip85Enabled, Block processingBlock, BlockHeader processingBlockHeader, Coin syntheticReward) {
-        RemascFederationProvider federationProvider = new RemascFederationProvider(activationConfig, constants.getBridgeConstants(), repository, processingBlock);
+
+        ActivationConfig.ForBlock activations = activationConfig.forBlock(processingBlock.getNumber());
+
```

Comment:
we could define 
`bridgeConstants = constants.getBridgeConstants()` and use the variable when needed